doc/basic_configuration.rst

```diff
@@ -55,17 +61,32 @@ In *all* build files::
     emails:
     - your_email@example.com
 
+Note that you need to have a local smtp service configured for email notifications.
+
+If you do not require email notifications, remove the configuration line entries.
+
 
 Update URLs to point to custom build farm
 -----------------------------------------
 
-Change the Jenkins URL in the entry point yaml to point to your earlier
-provisioned Jenkins master::
+Change the ``rosdistro_index_url`` in the `ros-infrastructure/ros_buildfarm_config <https://github.com/ros-infrastructure/ros_buildfarm_config>`_'s 
+``index.yaml`` to point to the ``rosdistro`` repository that defines the distro(s) that your buildfarm should build.
+
+**Important:** Note that this the `rosdistro <https://github.com/ros/rosdistro>`_'s ``index.yaml``
+(which points to the files that define which packages are included in the distro and respective caches),
+and **not** the ``index.yaml`` in ``ros_buildfarm_config`` which you are editing.
+
+This will usually be on the provisioned ``repo`` host, but can be the official `ros/rosdistro <https://github.com/ros/rosdistro>`_'s ``index.yaml`` if you intend to build the default set of packages::
+
+  rosdistro_index_url: http://repo_hostname.example.com/rosdistro/index.yaml
```

Comment:
Why would the `rosdistro` fork commonly be on the `repo` host? That sounds rather uncommon to me.

Reply:
Probably I was mistaken - I though ``ros_buildfarm_deployment`` automatically created folders ``rosdistro`` and ``rosdistro_cache`` on the ``repo`` host for this purpose. But now I'm not sure anymore.
Where do people usually host ``rosdistro`` if not on the repo host?

Reply:
> Where do people usually host `rosdistro` if not on the repo host?

On GitHub - same as the upstream repo.

Reply:
Thanks. I changed the wording accordingly.